src/types.cpp

```diff
@@ -22,6 +22,8 @@ limitations under the License.
 #include "anycellmlelement_p.h"
 #include "internaltypes.h"
 
+#include <limits>
+
```

Comment:
This system header should be higher in the include hierarchy.

Reply:
d955dd5a